src/hm/hmRadio.h

```diff
@@ -239,6 +240,14 @@ class HmRadio {
             return mNrf24.isPVariant();
         }
 
+        bool goodSignal(void) {
+            bool goodSignal = mNrf24.testRPD();
+            DPRINT(DBG_INFO, F("NRF Signal: "));
+            DPRINT(DBG_INFO, String(goodSignal));
+            mNrf24.read(0,0);
```

Comment:
was bedeutet diese Zeile?

Reply:
Die Zeile gibt offenbar ein good/bad (true/false) aus, das die Signalstärke (< oder > -64dBm) der einzelnen Inverter darstellt. Siehe Issue #1050

Reply:
das ist doch schon in Zeile 244. Mit geht es um Zeile 247

Reply:
Axo 😇

Reply:
Die Funktion wird aufgerufen sobald "Settings" aufgerufen wird. 
Das heißt es wird nicht jedesmal im Code abgefragt wie gut das Signal ist. 

"Macht man ja nur einmalig, damit man einen groben Wert hat."

245 + 246 -> Hupps, die können raus, war für mich zum debuggen noch. Jedoch hab ich mich wie ein Schnitzel gefreut das es direkt auf der Settings-Seite auch angezeigt wurde, das ich das so gesaved habe. :D Kann man raus nehmen, sollte man rausnehmen.